crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -223,7 +236,13 @@ impl AuthorityEpochTables {
 }
 
 impl AuthorityPerEpochStore {
-    pub fn new(committee: Committee, parent_path: &Path, db_options: Option<Options>) -> Arc<Self> {
+    pub fn new(
+        committee: Committee,
+        parent_path: &Path,
+        db_options: Option<Options>,
+        metrics: Arc<EpochMetrics>,
+    ) -> Arc<Self> {
```

Comment:
I think current_epoch should be set here as well. Otherwise restarted node will have current epoch at 0.